published/012-2016-August-24.md

```diff
@@ -0,0 +1,130 @@
+# IPFS Newsletter #11
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened since the IPFS team week in Lisbon in late July. This is the first Newsletter in a while, and there's a ton of exciting content below.
+
+For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [newsletter](//tinyletter.com/ipfsweekly)!
+
+## Updates
+
+### [Orbit](https://github.com/haadcode/orbit)
+
+![uPort](orbit-uport.gif)
+
+Orbit, the decentralized chat app run on top of IPFS, has just integrated uPort identities. There's a writeup on what uPort is [on Medium](https://medium.com/@ConsenSys/uport-the-wallet-is-the-new-browser-b133a83fe73) - essentially, it is a web-based wallet and identity management system. This means that users will be able to use their uPort name and profile picture in Orbit, as well as provision a signing key that will sign Orbit messages. We've been working with [ConsenSys](https://consensys.net/) to make this happen. You can track this [here](https://github.com/haadcode/orbit/issues/37). 
```

Comment:
If it's ok, we should bump the Devcon2 talks (add this at the end of the section):

"If you're going to Ethereum Foundation's [Devcon2](https://ethereumfoundation.org/devcon), @haadcode will dome the integration and talk about the implementation details on [21st of September](https://ethereumfoundation.org/devcon/?page_id=14) at the conference. Come see the talk!"